paasta_tools/autoscaling/cluster_boost.py

```diff
@@ -0,0 +1,195 @@
+"""
+Provides functions to temporary boost cluster capacity.
+Useful during big service bounce or failovers to preemptively increase capacity.
+
+This works by setting a temporary multiplier on the initial cluster's measured load.
+The resulting increased capacity is garanteed until the end of the boost.
+If usage gets higher the pool will behave normally and scale up
+This applies to any resource (cpu, memory, disk). Usually the limiting one is the cpu.
+Default duration of the boost factor is 40 minutes and default value is 1.5
+"""
+import logging
+from collections import namedtuple
+from datetime import datetime
+
+from kazoo.exceptions import NoNodeError
+
+from paasta_tools.utils import ZookeeperPool
+
+DEFAULT_BOOST_FACTOR = 1.5
+DEFAULT_BOOST_DURATION = 40
+
+MIN_BOOST_FACTOR = 1.0
+MAX_BOOST_FACTOR = 3.0
+
+MAX_BOOST_DURATION = 240
+
+
+BoostValues = namedtuple(
+    'BoostValues',
+    [
+        'end_time',
+        'boost_factor',
+        'expected_load',
```

Comment:
I'm trying to understand this value. It exists because you want the boost to just be `boost_factor` * `load_when_boost_first_runs`? Why not just make the boost always multiply by the current load?

Reply:
You understand right. We had a discussion about that and if we kept applying a boost multiplier to a boosted load and so on, we could end up scaling up way too much